Firestore/Example/Tests/API/FIRGeoPointTests.m

```diff
@@ -28,27 +29,24 @@ @interface FIRGeoPointTests : XCTestCase
 @implementation FIRGeoPointTests
 
 - (void)testEquals {
-  XCTAssertEqualObjects([[FIRGeoPoint alloc] initWithLatitude:0 longitude:0],
-                        [[FIRGeoPoint alloc] initWithLatitude:0 longitude:0]);
-  XCTAssertEqualObjects([[FIRGeoPoint alloc] initWithLatitude:1.23 longitude:4.56],
-                        [[FIRGeoPoint alloc] initWithLatitude:1.23 longitude:4.56]);
-  XCTAssertNotEqualObjects([[FIRGeoPoint alloc] initWithLatitude:0 longitude:0],
-                           [[FIRGeoPoint alloc] initWithLatitude:1 longitude:0]);
-  XCTAssertNotEqualObjects([[FIRGeoPoint alloc] initWithLatitude:0 longitude:0],
-                           [[FIRGeoPoint alloc] initWithLatitude:0 longitude:1]);
-  XCTAssertNotEqualObjects([[FIRGeoPoint alloc] initWithLatitude:0 longitude:0],
-                           [[NSObject alloc] init]);
+  FIRGeoPoint *foo = FSTTestGeoPoint(0, 0);
+  FIRGeoPoint *fooDup = FSTTestGeoPoint(0, 0);
+  FIRGeoPoint *bar = FSTTestGeoPoint(1.23, 4.56);
+  FIRGeoPoint *barDup = FSTTestGeoPoint(1.23, 4.56);
```

Comment:
I'm not sure bar / barDup is adding anything to these tests.  You do the same equals checks as with foo / fooDup. If you want non-zero values, I'd just change foo to be non-zero values.